### ISSUE TITLE
Footprint '* duckyb-collection:***' not found

### ISSUE THREAD

**Issue by Wesztman** [duckyb/urchin#4]
Hi! Love your work here <3 I have the Sweep myself and wanted to build a Urchin as well, but as my first ever keyboard modification I'm trying to make it a 40 key layout as I saw you had 3 missing keys in the matrix 😊

- One more thumb key
- One more column with two pinky keys

![image](https://user-images.githubusercontent.com/54413402/227732939-06106dad-912b-4b0e-9449-4a178c0635bc.png)

Now I should say that I'm a full blown KiCAD noob, but I'm getting errors that I believe indicates that I'm missing a library. Is this something you have shared somewhere? 😇

![image](https://user-images.githubusercontent.com/54413402/227733036-0eeaf2e1-bd80-44b2-8859-e981967dbf04.png)

**Comment by duckyb** [comment]
This is not really a problem, it's just kicad letting you know that it's missing the footprint library that I used. However the footprints on the PCB still load correctly so it doesn't cause any real issues.

However I took the time to export all of the footprints into a new library that is now included with the project.
If you pull the changes I just made the error should go away.

PS. Thank you so much for the donation! 🍻
